### ISSUE TITLE
[Enhancement]: Update the copy to end a free trial

### ISSUE THREAD

**Issue by warrendholmes** [Automattic/wc-calypso-bridge#1170]
### Describe the solution you'd like

A user can remove their plan anytime during the free trial by going to Upgrades -> Purchases -> Remove plan :

<img width="1101" alt="Screenshot 2023-05-31 at 13 30 10" src="https://github.com/Automattic/wc-calypso-bridge/assets/995473/7626aa7e-bb24-46b8-9336-5b95bd2bc916">

We should make it more clear that they'll be ending their free trial. Instead of "Remove plan" we could update the text to something like "End Free Trial".

@lauroraa Would you be able to suggest a more clear change here?

**Comment by lauroraa** [comment]
@warrendholmes Absolutely! If there's space, I would recommend combining the two to say:

Remove your plan and end free trial

Just because "End free trial" by itself might make people think this is what they need to do if they want to upgrade early.

**Comment by warrendholmes** [comment]
@lauroraa Thank you!
